src/utils/base64.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @function getIconAsBase64Uri
+ * @param {string} icon name
+ * @returns {string} base64-encoded data uri for SVG icon
+ */
+export const getIconAsBase64Uri = name => {
+	try {
+		return require(`base64-image!swarm-icons/dist/optimized/${name}.svg`);
+	} catch(e) {
+		return new Error(`Could not construct base64 uri for icon "${name}" \n ${e}`);
```

Comment:
does this `catch` work in practice? since the base-64 image is created by webpack at build time, the build should fail in case of an error, but this doesn't `throw` the error so I think it's probably not doing what it's intended to do at runtime